php/commands/site.php

```diff
@@ -121,8 +121,9 @@ public function code($args, $assoc_args) {
           Terminus::confirm('Commit %s changes?', $assoc_args, array($count));
```

Comment:
`$count` here is always 1 because `$diff` is an object instead of an array. `$count = count((array)$diff);` should fix this.

Also $count represents the number of files which have changed, not the number of changes. We should either add up the individual changes to each file or change the message to 'Commit changes to %s files?'.

We should also probably exit with a warning/info if there are no changes to commit rather than offer: 'commit 0 changes?'. Or if there's a meaningful reason to commit no changes we should say: 'There are no changed files. Commit anyway?'